script.js

```diff
@@ -0,0 +1,80 @@
+let turn = "X";
+let isGameOver = false;
+const boxes = document.getElementsByClassName("box");
+const resetBtn = document.getElementById("reset");
+const boxText = document.getElementsByClassName("text");
+let player = "Player 1";
+let isDraw = false;
+
+const changeTurn = () => {
+  return turn === "X" ? "0" : "X";
+};
+
+const changePlayer = () => {
+  return turn === "X" ? "Player 1" : "Plyer 2";
+};
+
+const checkWin = () => {
+  let wins = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6],
+  ];
+  wins.forEach((e) => {
```

Comment:
can we add comment here?, like what this matrix represent and and entity inside matrix